tests/aot/globals_test.py

```diff
@@ -242,9 +216,7 @@ def testTensorUpdateGlobalReturnNone(self):
         update_example = torch.randn(1, 1, 4)
 
         class UpdateState(CompiledModule):
-            state0 = export_global(
-                state_example, mutable=True, initialize=False
-            )
+            state0 = export_global(state_example, mutable=True, initialize=False)
```

Comment:
looks like we're using different black commands? Probably need a contributing.md

Reply:
Can you add an issue here. Should probably just do something like what IREE is doing.